Source/FullScreenPokemon.ts

```diff
@@ -1403,6 +1403,111 @@ module FullScreenPokemon {
             }
         }
 
+        /**
+         * Starts the Player fishing.
+         *
+         * @param player   A Player to start fishing.
+         * @todo Add a parameter and functionality to handle the different rods.
+         */
+        startFishing(player: IPlayer): void {
+            if (player.bordering[player.direction] === undefined ||
+                player.bordering[player.direction].title.indexOf("WaterEdge") === -1) {
+                player.FSP.cannotDoThat(player);
+                return;
+            }
+
+            player.FSP.MenuGrapher.createMenu("GeneralText", {
+                "deleteOnFinish": true,
+                "ignoreA": true,
+                "ignoreB": true
+            });
+            player.FSP.MenuGrapher.addMenuDialog(
+                "GeneralText",
+                [
+                    "%%%%%%%PLAYER%%%%%%% used Old Rod!"
+                ]);
+            player.FSP.MenuGrapher.setActiveMenu("GeneralText");
+
+            player.FSP.setWidth(player, 7, true, true);
```

Comment:
When they stop fishing the width and class is changed back to normal in the catchFish or noFish functions. If i put those statements in the startFishing function the class is removed too early.